pkg/api/scylla/validation/cluster_validation_test.go

```diff
@@ -87,6 +87,425 @@ func TestValidateScyllaCluster(t *testing.T) {
+			cluster: func() *v1.ScyllaCluster {
+				cluster := validCluster.DeepCopy()
+				cluster.Spec.Backups = append(cluster.Spec.Backups, v1.BackupTaskSpec{
+					SchedulerTaskSpec: v1.SchedulerTaskSpec{
+						Name: "task-name",
+						Cron: pointer.Ptr("CRON_TZ=Europe/Warsaw 0 23 * * SAT"),
+					},
+				})
+				return cluster
+			}(),
+			expectedErrorList: field.ErrorList{
+				&field.Error{
+					Type:     field.ErrorTypeInvalid,
+					Field:    "spec.backups[0].cron",
+					BadValue: pointer.Ptr("CRON_TZ=Europe/Warsaw 0 23 * * SAT"),
+					Detail:   "can't use TZ or CRON_TZ in cron, use timezone instead",
+				},
+			},
+			expectedErrorString: `spec.backups[0].cron: Invalid value: "CRON_TZ=Europe/Warsaw 0 23 * * SAT": can't use TZ or CRON_TZ in cron, use timezone instead`,
+		},
+		{
+			name: "invalid timezone in manager repair task spec",
+			cluster: func() *v1.ScyllaCluster {
+				cluster := validCluster.DeepCopy()
+				cluster.Spec.Repairs = append(cluster.Spec.Repairs, v1.RepairTaskSpec{
+					SchedulerTaskSpec: v1.SchedulerTaskSpec{
+						Name:     "task-name",
+						Cron:     pointer.Ptr("0 23 * * SAT"),
+						Timezone: pointer.Ptr("invalid"),
+					},
+					Intensity: "1",
+				})
+				return cluster
+			}(),
+			expectedErrorList: field.ErrorList{
+				&field.Error{
+					Type:     field.ErrorTypeInvalid,
+					Field:    "spec.repairs[0].timezone",
+					BadValue: pointer.Ptr("invalid"),
+					Detail:   "unknown time zone invalid",
+				},
+			},
+			expectedErrorString: `spec.repairs[0].timezone: Invalid value: "invalid": unknown time zone invalid`,
+		},
+		{
+			name: "invalid timezone in manager backup task spec",
+			cluster: func() *v1.ScyllaCluster {
+				cluster := validCluster.DeepCopy()
+				cluster.Spec.Backups = append(cluster.Spec.Backups, v1.BackupTaskSpec{
+					SchedulerTaskSpec: v1.SchedulerTaskSpec{
+						Name:     "task-name",
+						Cron:     pointer.Ptr("0 23 * * SAT"),
+						Timezone: pointer.Ptr("invalid"),
+					},
+				})
+				return cluster
+			}(),
+			expectedErrorList: field.ErrorList{
+				&field.Error{
+					Type:     field.ErrorTypeInvalid,
+					Field:    "spec.backups[0].timezone",
+					BadValue: pointer.Ptr("invalid"),
+					Detail:   "unknown time zone invalid",
+				},
+			},
+			expectedErrorString: `spec.backups[0].timezone: Invalid value: "invalid": unknown time zone invalid`,
+		},
+		{
+			name: "cron and timezone in manager repair task spec",
+			cluster: func() *v1.ScyllaCluster {
+				cluster := validCluster.DeepCopy()
+				cluster.Spec.Repairs = append(cluster.Spec.Repairs, v1.RepairTaskSpec{
+					SchedulerTaskSpec: v1.SchedulerTaskSpec{
+						Name:     "task-name",
+						Cron:     pointer.Ptr("0 23 * * SAT"),
+						Timezone: pointer.Ptr("Europe/Warsaw"),
+					},
+					Intensity: "1",
+				})
+				return cluster
+			}(),
+			expectedErrorList:   field.ErrorList{},
+			expectedErrorString: "",
+		},
+		{
+			name: "invalid timezone in manager backup task spec",
```

Comment:
it's unclear whether the test should produce and error or not from it's name (in other places as well), especially given it says `invalid` and produces no error

Reply:
my bad, it shouldn't say invalid here

Reply:
done